src/inverseRobot.js

```diff
@@ -24,7 +24,14 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = Object.entries(robot).map(([key, value]) => [value, key]);
```

Comment:
Nice idea of swapping, but constant `obj` is not an regular object, isn't it?) Better name it as `arrayEntries` or even `robotEntries`.